react/components/Conditions/index.tsx

```diff
@@ -129,6 +130,51 @@ const Conditions: React.FC<Props> = ({
                 {}
               )
 
+              const statementContent = [
+                <div className="flex-grow-1">
+                  <Statement
+                    isRtl={isRtl}
+                    isFullWidth={isFullWidth}
+                    onChangeStatement={newStatement => {
```

Comment:
Nitpick, when complexity is increasing, moving it to an external function helps readability.

Reply:
Done! looks nice